internal/validators/tests/validators-specs.js

```diff
@@ -0,0 +1,126 @@
+import {
```

Comment:
Would you please refer to https://github.com/enactjs/enact/pull/3090/files#diff-1f7ad7cfd88d5c0f064d6e2d65af1461e72ea3388f9bf1206d22854ba419a688?
I've already reviewed enact's validators. I think it would be the same tests.